src/components/molecules/PaginationButtons/index.tsx

```diff
@@ -2,10 +2,15 @@ import ReactPaginate from 'react-paginate'
 
 import { PaginationButtonProps } from './PaginationButtons.types'
 
-const PaginationButtons = ({ pageCount, onChange }: PaginationButtonProps) => (
+const PaginationButtons = ({
+  pageCount,
+  onChange,
+  forcePage,
+}: PaginationButtonProps) => (
   <ReactPaginate
     pageCount={pageCount}
     pageRangeDisplayed={3}
+    forcePage={forcePage - 1}
```

Comment:
Is it just a parameter to handle paging on our own?

Reply:
Kind of- it helped me with keeping proper pagination page when I go back from ProductDetails page to ProductList. If I didn't use it- every time I went back I was on 1st page, even if product I clicked on was on page 4.